### PR TITLE
Redis Commaner with configurable background

### PR DESCRIPTION
**Pull request by Sawan88**

### CODE REVIEW

File: .github/workflows/docker-image-snapshot-branch.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # See https://github.com/docker/build-push-action#readme
+      # See https://github.com/docker/build-push-action/blob/b1aeb1103e6b3b5648dbd6deaf0559919456ca6b/docs/advanced/tags-labels.md
+      - name: Build And Push
+        id: docker_build
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          platforms: linux/arm64,linux/amd64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6
```
Comment (theanurin) [inline .github/workflows/docker-image-snapshot-branch.yml]: Looks like you need to remove

- linux/arm64
- linux/ppc64le
- linux/s390x
- linux/386
- linux/arm/v7
- linux/arm/v6

File: README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+[![Docker Image Version](https://img.shields.io/docker/v/sergii.ivanov/redis-commander?sort=date&label=Version)](https://hub.docker.com/r/sergii.ivanov/redis-commander/tags)
```
Comment (theanurin) [inline README.md]: Pls replace `sergii.ivanov` -> `theanurin`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# Spec
+
+## Environment variables
+
+No any variables
+
+## Expose ports
+
+* `tcp/39000` - redis-commander listening endpoint
```
Comment (theanurin) [inline README.md]: Pls replace `tcp/39000` -> `tcp/8081`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+# Image reason
+
+No special any reason. This just our infrastructure image.
+
+# Spec
+
+## Environment variables
+
+No any variables
```
Comment (theanurin) [inline README.md]: Pls add definition of `REDIS_COMMANDER_BG_COLOR`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## Expose ports
+
+* `tcp/39000` - redis-commander listening endpoint
+
+## Volumes
+
+* `/docker-images` - Root of the redis-commander repositories
```
Comment (theanurin) [inline README.md]: Pls write

No any volumes

File: README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+# Inside
+
+* redis-commander/redis-commander:19cd0c49f418
```
Comment (theanurin) [inline README.md]: Here we have to define Redis Commander version (not image)

File: docker/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,11 @@
+FROM rediscommander/redis-commander@sha256:19cd0c49f418779fa2822a0496c5e6516d0c792effc39ed20089e6268477e40a
+
+USER root
+
+RUN apk add --no-cache sudo
+COPY docker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint-redis-commander.sh
+
+USER redis
+
```
Comment (theanurin) [inline docker/Dockerfile]: Pls add
```
ENV REDIS_COMMANDER_BG_COLOR=
```
See more https://docs.docker.com/engine/reference/builder/#env

File: docker/docker-entrypoint.sh [hunk 8]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+set -e
+
+apk add sudo
```
Comment (theanurin) [inline docker/docker-entrypoint.sh]: Pls remove this row.
`sudo` is installed at image build time

**Review by theanurin — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-image-release.yml [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Update image README and description
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          repository: ${{ secrets.DOCKERHUB_USERNAME }}/redis-commander
+          short-description: "Apache Redis-commander is a software versioning and revision control system distributed as open source under the Apache License."
```
Comment (theanurin) [inline .github/workflows/docker-image-release.yml]: Incorrect `short-description`

**Review by theanurin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Sawan88) [reply README.md]: fix

File: README.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+[![Docker Pulls](https://img.shields.io/docker/pulls/sergii.ivanov/redis-commander?label=Pulls)](https://hub.docker.com/r/sergii.ivanov/redis-commander)
+[![Docker Stars](https://img.shields.io/docker/stars/sergii.ivanov/redis-commander?label=Docker%20Stars)](https://hub.docker.com/r/sergii.ivanov/redis-commander)
+
+# Redis Commander
+
+Redis web management tool written in node.js
+
+# Image reason
+
+No special any reason. This just our infrastructure image.
```
Comment (theanurin) [inline README.md]: Describe real reason of the image.

**Review by theanurin — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (Sawan88) [reply README.md]: * redis-commander/redis-commander:latest
DIGEST:sha256:19cd0c49f418779fa2822a0496c5e6516d0c792effc39ed20089e6268477e40a
or remove DIGEST?

File: .github/workflows/docker-image-snapshot-branch.yml (hunk 1 above)
Comment (Sawan88) [reply .github/workflows/docker-image-snapshot-branch.yml]: Fixes

File: README.md (hunk 4 above)
Comment (Sawan88) [reply README.md]: set the option color:
--env REDIS_COMMANDER_BG_COLOR=#......

this definition is correct?

maybe like this: set the value at the docker run
#ffffff - will make the color white

File: docker/Dockerfile (hunk 7 above)
Comment (Sawan88) [reply docker/Dockerfile]: Fixes

File: .github/workflows/docker-image-release.yml (hunk 9 above)
Comment (Sawan88) [reply .github/workflows/docker-image-release.yml]: Fixes

File: README.md (hunk 10 above)
Comment (Sawan88) [reply README.md]: Add reason of the image

**Pull request closed without merging**